server/game/cards/characters/04/eliasand.js

```diff
@@ -7,26 +7,20 @@ class EliaSand extends DrawCard {
                 afterChallenge: (event, challenge) => this.controller === challenge.loser
             },
             limit: ability.limit.perPhase(2),
-            handler: () => {
-                this.game.promptForSelect(this.controller, {
-                    activePromptTitle: 'Select character',
-                    source: this,
-                    cardCondition: card => card.location === 'play area' && card.getType() === 'character',
-                    onSelect: (p, card) => this.onCardSelected(p, card)
-                });
+            target: {
+                activePromptTitle: 'Select a character to gain stealth',
```

Comment:
I think we've been standardising on 'select a character' for stuff like this to not have the prompt get super long or wordy and keep it consistent